src/renderers/WebGLRenderer.js

```diff
@@ -2383,10 +2382,10 @@ class WebGLRenderer {
 
 				const reverseDepthBuffer = state.buffers.depth.getReversed();
 
-				if ( reverseDepthBuffer && camera.coordinateSystem !== ReversedCoordinateSystem ) {
+				if ( reverseDepthBuffer && camera.reversedDepth === false ) {
```

Comment:
`camera.reversedDepth !== true`

Reply:
Does that matter? `reversedDepth` as a boolean is defined for all types of camera.

Reply:
I believe so. It must be explicitly set to `true`, not `undefined` or `null` for example.

https://github.com/mrdoob/three.js/commit/7d3562fc04ee6fc501e9924ff3f48839d2122a97

Reply:
According to the JSDoc, `undefined` or `null` would not be valid values. So assigning these values wouldn't be correct in the first place.